src/main/java/com/cinema/security/CustomUserDetailsService.java

```diff
@@ -0,0 +1,36 @@
+package com.cinema.security;
+
+import com.cinema.model.Role;
+import com.cinema.model.User;
+import com.cinema.service.UserService;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.security.core.userdetails.UsernameNotFoundException;
+import org.springframework.stereotype.Service;
+
+@Service
+public class CustomUserDetailsService implements UserDetailsService {
+
+    private final UserService userService;
+
+    public CustomUserDetailsService(UserService userService) {
+        this.userService = userService;
+    }
+
+    @Override
+    public UserDetails loadUserByUsername(String email) throws UsernameNotFoundException {
+        User user = userService.findByEmail(email);
+        org.springframework.security.core.userdetails.User.UserBuilder builder;
```

Comment:
```suggestion
        UserBuilder builder;
```
спробуй перенести зайве в імпорти